Source/nthread.cpp

```diff
@@ -288,7 +288,7 @@ void __fastcall nthread_ignore_mutex(BOOL bStart)
 }
 // 67975A: using guessed type char sgbThreadIsRunning;
 
-BOOL __cdecl nthread_has_500ms_passed()
+BOOL __fastcall nthread_has_500ms_passed(int dummy)

```

Comment:
Personal preference, but I would rename `dummy` to `unused`.

Reply:
Your right `ùnused` is what they are named elsewhere in the project.